### ISSUE TITLE
BBQ - Go client and library for BoxBackup.

### ISSUE THREAD

**Issue by karinushka** [boxbackup/boxbackup#115]
I have "reverse-engineered" the BoxBackup protocol in Go and wrote a simple reference client:
https://github.com/karinushka/bbq

Check it out - using Go makes it more accessible to people and very easy to plug into existing frameworks, such as Go-Fuse and such.
Also the command line client has neater features than the existing `bbackupquery` :)

**Comment by qris** [comment]
(cc: the mailing list)

Hi Karinusha,

Thank you! I will link to this in the docs (I assume you don't want it
merged into contrib).

I'm sorry that you had to reverse-engineer the protocol. I think we failed
to sufficiently signpost the (rather sparse) protocol documentation here:
https://github.com/boxbackup/boxbackup/wiki/BoxBackupStoreProtocol

I've added a link to it directly in the TOC. Any help in filling it out
(from anyone) would be greatly appreciated! I will try to put more time
into it as soon as I can.

Thanks, Chris.
